common/src/main/webapp/less/style.less

```diff
@@ -1198,7 +1212,9 @@ img.partner-split-flag {
   margin-bottom: 0;
 }
 
-i.icon.user-astronaut::before { content: "\f4fb"; }
+i.icon.user-astronaut::before {
```

Comment:
Unrelated but I suspect we don't need this anymore with FontAwesome icons in react